lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -6834,11 +6834,19 @@ namespace Js
                     - If the size is >= 100, scale by 1.6
             */
             const uint loopPercentage = GetByteCodeInLoopCount() * 100 / max(1u, GetByteCodeCount());
-            if(loopPercentage <= 50)
+            const int byteCodeSizeThresholdForInlineCandidate = CONFIG_FLAG(InlineThreshold) + CONFIG_FLAG(InlineThresholdAdjustCountInSmallFunction);
+            bool delayFullJITThisFunc =
+                (CONFIG_FLAG(DelayFullJITSmallFunc) > 0) && (byteCodeSizeThresholdForInlineCandidate < 0 || this->GetByteCodeWithoutLDACount() > (uint)byteCodeSizeThresholdForInlineCandidate);
```

Comment:
Shouldn't the byte code size of current function be less than the byteCodeSizeThresholdForInlineCandidate for you to be sure that this function will get inlined?